src/main/java/core/basesyntax/ReportDateUtil.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class ReportDateUtil {
+    private static final String DATE_PATTERN = "dd.MM.yyyy";
+    private final LocalDate date;
+
+    public ReportDateUtil(String dateString) {
+        this.date = stringDateToLocalDate(dateString);
+    }
+
+    public static LocalDate stringDateToLocalDate(String date) {
+        return LocalDate.parse(date, DateTimeFormatter.ofPattern(DATE_PATTERN));
+    }
+
+    public boolean checkIsDateInRange(String rangeStart, String rangeEnd) {
+        LocalDate rangeStartDate = stringDateToLocalDate(rangeStart);
+        LocalDate rangeEndDate = stringDateToLocalDate(rangeEnd);
+        return (date.isAfter(rangeStartDate) || date.isEqual(rangeStartDate))
+                && (date.isBefore(rangeEndDate) || date.isEqual(rangeEndDate));
+    }
+}
```

Comment:
This is not a good approach.
1. Learn what util class is: https://www.baeldung.com/java-helper-vs-utility-classes#:~:text=A%20utility%20class%20in%20Java,sometimes%20stateless%20without%20static%20variables.

2. LocalDate.parse is enough for this functionality.